test/functional/tests/cli/test_cli_help_and_version.py

```diff
@@ -19,7 +18,7 @@
 def test_cli_help(shortcut):
     """
     title: Test for 'help' command.
-    description: Test if help for commands displays.
+    description: Test if help for commands displays correct output.
     pass_criteria:
       - Proper help displays for every command.
```

Comment:
I commentend on this in Kamil's PR https://github.com/Open-CAS/open-cas-linux/pull/1596, so hopefully this will be fixed there